### PR TITLE
Added ZeroSSL tutorial

### PR DESCRIPTION
**Pull request by czomo**
Signed-off-by: czomo <tomaszjdul@gmail.com>

### CODE REVIEW

**Comment by czomo** [comment]
I've added basic tutorial&troubleshooting with ZeroSSL. I think is worth to share knowledge that there are more solutions like Let's encrypt on the market. Cheers!

**Comment by czomo** [comment]
@irbekrm Can you look at this?

File: content/en/docs/tutorials/zerossl/zerossl.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+```
+```bash
+kubectl apply -f zero-sll-eabsecret.yaml -n cert-manager
+```
+
+### Second workaround of creating secret.
+
+```bash
+$ kubectl create secret generic \
+       zero-sll-eabsecret \
```
Comment (pascal-hofmann) [inline content/en/docs/tutorials/zerossl/zerossl.md:84]:
```suggestion
       zero-ssl-eabsecret \
```

File: content/en/docs/tutorials/zerossl/zerossl.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+  Conditions:
+    Last Transition Time:  2021-09-09T17:03:26Z
+    Message:               The ACME account was registered with the ACME server
+    Reason:                ACMEAccountRegistered
+    Status:                True
+    Type:                  Ready
+```
+
+### Please note!
+If this step failed and the ACME account is not registered please check if secrets in `zero-sll-eabsecret` are correct.
```
Comment (pascal-hofmann) [inline content/en/docs/tutorials/zerossl/zerossl.md:152]:
```suggestion
If this step failed and the ACME account is not registered please check if secrets in `zero-ssl-eabsecret` are correct.
```

File: content/en/docs/tutorials/zerossl/zerossl.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+    # name of a secret used to store the ACME account private key
+    privateKeySecretRef:
+      name: zerossl-prod
+
+    # for each cert-manager new EAB credencials are required
+    externalAccountBinding:
+      keyID: ZEROSSL_KEY_ID
+      keySecretRef:
+        name: zero-sll-eabsecret
```
Comment (pascal-hofmann) [inline content/en/docs/tutorials/zerossl/zerossl.md:111]:
```suggestion
        name: zero-ssl-eabsecret
```

**Review by pascal-hofmann — CHANGES_REQUESTED** [review]
Typos

File: content/en/docs/tutorials/zerossl/zerossl.md (hunk 3 above)
Comment (czomo) [reply content/en/docs/tutorials/zerossl/zerossl.md]: Thanks @pascal-hofmann for pointing this out!

**Review by jakexks — APPROVED** [review]
/lgtm

Thank you for writing this!

**Comment by czomo** [comment]
Sth is not right here @jakexks Are you sure I should put it under `content/v1.8-docs/` ?

**Pull request merged**